src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,89 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String NAME_SUPPLY = "supply";
+    private static final String NAME_BUY = "buy";
+    private static final int INDEX_BUY = 0;
+    private static final int INDEX_SUPPLY = 1;
+    private static final int INDEX_TOTAL = 2;
+    private static final int INDEX_STRING = 0;
+    private static final int INDEX_INT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] values = getValues(fromFileName);
+        StringBuilder finalStatistic = new StringBuilder();
+        finalStatistic.append(NAME_SUPPLY)
+                .append(",")
+                .append(values[INDEX_SUPPLY])
+                .append(System.lineSeparator())
+                .append(NAME_BUY)
+                .append(",")
+                .append(values[INDEX_BUY])
+                .append(System.lineSeparator())
+                .append("result")
+                .append(",")
+                .append(values[INDEX_TOTAL]);
+        String recordToFile = finalStatistic.toString();
+        writeFile(recordToFile, toFileName);
+    }
+
+    private static String readFile(String fromFileName) {
+        File allData = new File(fromFileName);
+        String fileToString;
+        try {
+            BufferedReader readFile = new BufferedReader(new FileReader(allData));
+            StringBuilder dataContainer = new StringBuilder();
+            int value = readFile.read();
+            while (value != -1) {
+                dataContainer.append((char) value);
+                value = readFile.read();
+            }
+            fileToString = dataContainer.toString();
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
+        }
+        return fileToString;
+    }
+
+    private int[] getValues(String fromFileName) {
+        int sumBuy = 0;
+        int sumSupply = 0;
+        int total = 0;
+        int temporaryValue = 0;
+        String temporaryValueName = "";
+        String fileToString = readFile(fromFileName);
+        String[] stringToArray = fileToString.split("\n");
```

Comment:
```suggestion
        String[] stringToArray = fileToString.split(System.lineSeparator());
```

Reply:
when i tried to use System.lineSeparator(). then string do not split correct. i tried to do this way:
Pattern pattern = Pattern.compile(System.lineSeparator());
 String[] stringToArray = pattern .split(fileToString);
 but it also did not work.
 that's why I did String[] stringToArray = fileToString.split("\\r?\\n"); in next commit